articles/overview/overview.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Introduction to quantum computing
+description: Learn what quantum computing is, what quantum computers can do, and how you can learn quantum computing.
+author: bradben
+ms.author:  bradben
+ms.date: 04/22/2020
+ms.topic: overview
+uid: microsoft.quantum.overview.introduction
+---
+
+# Introduction to quantum computing 
+
+There are some problems so difficult, so incredibly vast, that even if every supercomputer in the world worked on the problem, it would still take longer than the lifetime of the universe to solve.
```

Comment:
This feels a bit over the top; I'd suggest toning it down a bit, especially for an opening paragraph.

Reply:
Agree. Tone it down. Something like: for certain tasks, our most powerful computers run into limitations. 
BTW, would you like to use my cartoons in this page? [Kitty's Hackaday comics for hobbyists](https://hackaday.io/project/168554/logs?sort=newest&page=2)